libs/designsystem/src/lib/components/list/list.component.scss

```diff
@@ -160,10 +159,29 @@ ion-item-options[side='end'] {
     border-radius: $border-radius;
   }
 
+  .virtual-with-sections-wrapper {
+    margin-left: -1 * size('xxxs');
+    margin-right: -1 * size('xxxs');
+  }
+
```

Comment:
what is the reason for adding this? It is added in a commit that supposedly should fix some cookbook stuff, but it changes the list component itself.

Reply:
Seems like the reason is for the box-shadow to render correctly. But we need to find a better strategy for this, as this way will play weird with 100% width layouts - e.g. in a popup-example, we get weird horizontal overflow when this is present.